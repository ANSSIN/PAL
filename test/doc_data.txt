PdfID0: a77b59804ccb882104cc9ed85f2dc2c2
PdfID1: a77b59804ccb882104cc9ed85f2dc2c2
NumberOfPages: 2
PageMediaBegin
PageMediaNumber: 1
PageMediaRotation: 0
PageMediaRect: 0 0 612 792
PageMediaDimensions: 612 792
PageMediaBegin
PageMediaNumber: 2
PageMediaRotation: 0
PageMediaRect: 0 0 612 792
PageMediaDimensions: 612 792
