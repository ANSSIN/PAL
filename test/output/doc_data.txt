PdfID0: a7b7f97761c574bad59e479b9e95fa65
PdfID1: a7b7f97761c574bad59e479b9e95fa65
NumberOfPages: 1
PageMediaBegin
PageMediaNumber: 1
PageMediaRotation: 0
PageMediaRect: 0 0 612 792
PageMediaDimensions: 612 792
