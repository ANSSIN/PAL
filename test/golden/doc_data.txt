PdfID0: 834325f6bf21bf1f685cae256aa97447
PdfID1: 834325f6bf21bf1f685cae256aa97447
NumberOfPages: 1
PageMediaBegin
PageMediaNumber: 1
PageMediaRotation: 0
PageMediaRect: 0 0 612 792
PageMediaDimensions: 612 792
